### PR TITLE
Added GizmoPool

### PR DESCRIPTION
**Pull request by jangid**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3248,5 +3248,20 @@
...
+    "poolId": "b8e25baf77c6387d155cc2f11b1ba4cc652c697e8eb4bf41755b311f",
+    "social": {
+      "facebook": "",
+      "twitter": "",
+      "website": "https://gizmopool.com",
+      "reddit": "GizmoPool",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "GizmoPool",
```
Comment (Bez625) [inline registry.json:3263]: Please can you change this to an individual TG handle and not a group or channel?

File: registry.json [hunk 2]
```diff
@@ -3248,5 +3248,20 @@
...
     "operatorName": "George",
     "github": "cardano-apexpool"
+  },
+  {
+    "ticker": "GIZMO",
+    "poolId": "b8e25baf77c6387d155cc2f11b1ba4cc652c697e8eb4bf41755b311f",
+    "social": {
+      "facebook": "",
+      "twitter": "",
+      "website": "https://gizmopool.com",
```
Comment (Bez625) [inline registry.json:3258]: This website doesnt seem to work... I get an unauthorised

File: registry.json [hunk 3]
```diff
@@ -3248,5 +3248,20 @@
...
+      "reddit": "GizmoPool",
```
Comment (Bez625) [inline registry.json:3259]: This user doesnt seem to exist on reddiut

File: registry.json [hunk 4]
```diff
@@ -3248,5 +3248,20 @@
     "telegram": "apex_pool_spo",
     "operatorName": "George",
     "github": "cardano-apexpool"
+  },
+  {
+    "ticker": "GIZMO",
+    "poolId": "b8e25baf77c6387d155cc2f11b1ba4cc652c697e8eb4bf41755b311f",
+    "social": {
+      "facebook": "",
```
Comment (Bez625) [inline registry.json:3256]: Please can you add some form of social media presence? Generally people choose twitter

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by jangid** [comment]
I have made the suggested changes.

**Pull request merged**